### DISCUSSION TITLE
draft pull request disappeared

### DISCUSSION THREAD

**Discussion by kdraeder** [ESMCI/cime#4450] [Q&A]
I created a draft PR using the botton at the bottom of the "new PR" page,
which gives a choice between a regular and draft PRs.  Things looked fine.
Then I assigned the PR, entered a reviewer's name, and added some labels.
I reloaded the page and my PR disappeared.  I can't see it in the list of all open PRs,
and have had no success finding it.  Any suggestions?

**Comment by gold2718** [comment]
What was it about? Can you say anything about the title or content?

**Reply by kdraeder** [reply]
It's possible that I didn't enter a title.  Would that be fatal?
The content was about updating st_archive to handle new file types from DART.

**Reply by kdraeder** [reply]
I tried opening a draft PR on another (test) github site, both with and without a title,
and neither one showed up in the list of PRs.   So maybe it's something special about draft PRs. 
I should be able to easily see my own, though. 

What's the advantage of a draft PR over an issue, which I thought was meant to enable the same thing;
discussing something before implementing a fix?

**Comment by gold2718** [comment]
> What's the advantage of a draft PR over an issue, which I thought was meant to enable the same thing; discussing something before implementing a fix?

The draft PR is for when you have code changes to discuss. An issue is more useful before codng happens.

**Comment by gold2718** [comment]
@kdraeder, can you give an approximate time you made the last action on the 'disappeared' PR?
Can an owner of ESMCI check the audit log for around that time to see if anything shows up?

**Reply by kdraeder** [reply]
Best guess; 11:30 a.m. - noon.
Thanks for bringing up this possibility.
But it might not be worth digging too deeply, since the same thing happens on a different repository,
and I've opened an issue to replace the lost draft PR.

**Comment by kdraeder** [comment]
Helen Kershaw led me to enlightenment.
It turns out that I didn't recognize that the green button at the bottom of the new PR page
is actually 2 buttons; "create pull request" and a drop down menu to choose draft PR.
I had selected draft PR, but failed to then click the "create draft pull request".
The content that I had entered was in the cache, but not yet in github.
If I had tried to open a second new PR in the same browser tab, I would have seen all of the content
that I thought was missing.
